cscan.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 pushd "$(dirname ${BASH_SOURCE[0]})" > /dev/null
 if [ ! -d ./tlslite ]; then
-    git clone --depth=1 https://github.com/tomato42/tlslite-ng.git .tlslite-ng
+    git clone --depth=1 https://github.com/tomato42/tlslite-ng.git .tlslite-ng &>/dev/null
```

Comment:
I think we can hardcode it for now, or just add a variable with `cscan.sh` in it